package.json

```diff
@@ -28,7 +28,9 @@
     "lib/",
     "!lib/**/*.test.js"
   ],
-  "dependencies": {},
+  "dependencies": {
+    "array-from": "2.1.1"
```

Comment:
Did you add this with the exact version number on purpose?

Reply:
Nope